### PR TITLE
Sync 1.18 updates

### PR DESCRIPTION
**Pull request by dhaiducek**

### CODE REVIEW

**Review by JustinKuli — APPROVED** [review]
Did magic mirror just miss these earlier?

File: Makefile [hunk 1]
```diff
@@ -226,7 +226,7 @@ kind-bootstrap-cluster-dev: kind-create-cluster install-crds install-resources
 .PHONY: kind-deploy-controller
 kind-deploy-controller: generate-operator-yaml
 	@echo installing $(IMG)
-	kubectl create ns $(KIND_NAMESPACE) || true
+	kubectl create ns $(KIND_NAMESPACE)
```
Comment (JustinKuli) [inline Makefile]: Might need `-` at start of the command. But we'd want to add that upstream.

**Review by JustinKuli — APPROVED** [review]
I think it needs either overrides or prow updates (or both)

**Comment by dhaiducek** [comment]
No overrides. Prow has been updated, so we should only need to wait for the branch protection to update:
- https://github.com/openshift/release/pull/30772

**Pull request merged**